benchmarks/experiment_runner.py

```diff
@@ -208,8 +222,8 @@ def _fwd_captured_stdout_stderr(self, stdout_text: str, stderr_text: str):
     print(stdout_text, file=sys.stdout, end='', flush=True)
     print(stderr_text, file=sys.stderr, end='', flush=True)
 
-  def _default_iter_fn(self, benchmark_experiment, benchmark_model,
-                       input_tensor):
+  def _default_iter_fn(self, benchmark_experiment: BenchmarkExperiment,
+                       benchmark_model: BenchmarkModel, input_tensor):
```

Comment:
Missing type for `input_tensor`.

Reply:
@zpcore

Reply:
Oh, sorry I didn't notice the comment. Let me address them in another PR.